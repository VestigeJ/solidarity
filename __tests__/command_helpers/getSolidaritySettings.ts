import getSolidaritySettings from '../../src/extensions/functions/getSolidaritySettings'
import context from 'gluegun'

test('getSolidaritySettings exists', () => expect(getSolidaritySettings).toMatchSnapshot())

test('getSolidaritySettings succeeds', async () => {
  const resultSettings = getSolidaritySettings(context)
  // we got an object with requirements defined
  expect(resultSettings).toMatchObject({requirements: {}})
})

test('getSolidaritySettings can fail', async () => {
  expect(() => {
    process.chdir('__tests__')
    const resultSettings = getSolidaritySettings(context)
  }).toThrow()
  process.chdir('../')
})
