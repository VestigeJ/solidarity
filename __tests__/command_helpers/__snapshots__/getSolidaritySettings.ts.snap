// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getSolidaritySettings exists 1`] = `[Function]`;
